lib/FrontendTool/FrontendTool.cpp

```diff
@@ -1082,6 +1082,18 @@ static void performSILOptimizations(CompilerInvocation &Invocation,
   }
 }
 
+/// Get the main source file's private discriminator and attach it to
+/// the compile unit's flags.
+static void setMSFPrivateDiscriminator(IRGenOptions &IRGenOpts,
```

Comment:
"MSF" again, but really this already doesn't work in WMO mode and won't work in batch mode. @adrian-prantl, are we still using this at all?

Reply:
Changing it to setPrivateDiscrimatorIfNeeded, and leaving the functionality in for now. (Don't want to get sidetracked.)